xds/internal/balancer/edsbalancer/eds_test.go

```diff
@@ -108,10 +108,12 @@ func (f *fakeEDSBalancer) handleChildPolicy(name string, config json.RawMessage)
 	f.childPolicy.Send(&loadBalancingConfig{Name: name, Config: config})
 }
 
-func (f *fakeEDSBalancer) close()                                              {}
 func (f *fakeEDSBalancer) handleEDSResponse(edsResp xdsclient.EndpointsUpdate) {}
+
 func (f *fakeEDSBalancer) updateState(priority priorityType, s balancer.State) {}
 
+func (f *fakeEDSBalancer) close() {}
```

Comment:
Embed `edsBalancerImplInterface` in `fakeEDSBalancer` instead and get rid of these empty methods?

Reply:
Good call. Done

Reply:
This is reverted. It turns out those methods can be noop, but cannot be nil...